File: modulo4_memoria/aula/phonebook.c
#include<cs50.h>
#include<stdio.h>
#include<string.h>

int main(void){

  FILE *file= fopen("phonebook.csv","a");

  if(file==NULL){
    return 1;
  }

  char *name=get_string("Name: ");
  char *number=get_string("Number: ");

  fprintf(file,"%s,%s\n",name,number);

  fclose(file);
}

File: modulo6_python/aula/phonebook.py
import cs50
import csv

def main():

    file= open("phonebook.csv","a")

    name=cs50.get_string("Name: ")
    number=cs50.get_string("Phone: ")

    writer=csv.writer(file)

    writer.writerow([name,number])

    file.close()

main()